<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddComputerName</class>
 <widget class="QDialog" name="AddComputerName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>423</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_computerName">
         <property name="text">
          <string>Имя ПК</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_description">
         <property name="text">
          <string>Описание</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_imagePath">
         <property name="text">
          <string>Путь до картинки</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit_computerName"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_description"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_imagePath"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="button_addComputerName">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
